package.json

```diff
@@ -5,7 +5,10 @@
   "main": "out/index.js",
   "scripts": {
     "test": "mocha --exit -r ts-node/register test/*.spec.ts",
-    "build": "npx tsc"
+    "build": "npx tsc",
+    "pretest": "npm run prepare",
+    "prepare": "npm run lint && npm run build",
```

Comment:
The linting should be run as part of testing, not as part of the prepare script (it's better for the prepare script to just handle building, if you `npm install git://...` the module you don't care if the lint passes or not just that you have the built files to run)

Reply:
gotcha, just copied what [sbvr-compiler](https://github.com/balena-io-modules/sbvr-compiler/blob/master/package.json) does, but that makes more sense

Reply:
Yeah, that's not a great one to copy because it was a hack friday project as opposed to a more maintained up-to-date project so is fairly outdated on our coding/module standards (ie it predates `npm prepare` being a thing and I only just updated it to at least fix that much since otherwise it'll fail to publish on newer npm versions)